### PR TITLE
Issue 221

### PR DESCRIPTION
**Pull request by elma13**
confidentbands and variation in plots

### CODE REVIEW

**Comment by elma13** [comment]
@jspijker , for the text in the json, I only changed the Dutch text. Please feel free to translate in English. First only the expl, small text sentences.

File: funs/data_to_tool_fun.R [hunk 1]
```diff
@@ -89,3 +89,102 @@ add_uncertainty_sensor <- function(data_measurements, uc_pm10 = 8.5, uc_pm25 = 5
...
+  #   uc_pm10: double, to indicate the percentage uncertainty in pm10, default = 20
+  #   uc_pm25: double, to indicate the percentage uncertainty in pm25, default = 20
+  #   uc_pm10: double, to indicate the minimal uncertainty in pm10, default = 8.5
+  #   uc_pm25: double, to indicate the minimal uncertainty in pm25, default = 5.3
+  # return: dataframe as data_measurements with extra column "sd",
+  # this column is 0 for reference stations
+
+  # Add uncertainty to the measurements of the sensors
+  data_measurements <- data_measurements %>%
+    dplyr::mutate(sd = dplyr::case_when(parameter == "pm25_kal" & !grepl("NL", station) ~ (uc_pm25/100*value),
```
Comment (jspijker) [inline funs/data_to_tool_fun.R:109]: This assumes that the string "NL" never occurs in the name of a sensor/station. I'm not sure about this. Is it possible to add a more restrictive regex, like "^NL.*[0-9].*" (from the top of my head, use at your own risk ;-)

**Review by jspijker — APPROVED** [review]

**Comment by elma13** [comment]
> @elma13 : you removed plotly? Can we remove the library also from the container?

@jspijker , we don't use plotly at the moment, so yes it can be removed from the container

File: funs/data_to_tool_fun.R (hunk 1 above)
Comment (elma13) [reply funs/data_to_tool_fun.R]: See commit

**Pull request merged**